<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PodcastSettingsPage</class>
 <widget class="QWidget" name="PodcastSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Podcasts</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Updating</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Check for new episodes</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="check_interval">
        <item>
         <property name="text">
          <string>Manually</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Every 10 minutes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Every 20 minutes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Every 30 minutes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Every hour</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Every 2 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Every 6 hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Every 12 hours</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Download episodes to</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="auto_download">
        <property name="text">
         <string>Download new episodes automatically</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="download_dir"/>
        </item>
        <item>
         <widget class="QPushButton" name="download_dir_browse">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Cleaning up</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Delete played episodes</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="delete_after">
        <property name="specialValueText">
         <string>Manually</string>
        </property>
        <property name="suffix">
         <string> days</string>
        </property>
        <property name="prefix">
         <string>After </string>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="empty_text" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Appearance</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="hide_listened">
        <property name="text">
         <string>Don't show listened episodes</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="show_episodes">
        <property name="specialValueText">
         <string>All</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Number of episodes to show</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>gpodder.net</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Clementine can synchronize your subscription list with your other computers and podcast applications.  &lt;a href=&quot;https://gpodder.net/register/&quot;&gt;Create an account&lt;/a&gt;.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="LoginStateWidget" name="login_state" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="login_group" native="true">
        <layout class="QFormLayout" name="formLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Username</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="username"/>
           </item>
           <item>
            <widget class="QPushButton" name="login">
             <property name="text">
              <string>Sign in</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="password">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Device name</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="device_name"/>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="gpodder_advanced">
           <property name="text">
            <string>Advanced Settings</string>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QGroupBox" name="gpodder_advanced_group">
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Base URL</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="gpodder_base_url"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LoginStateWidget</class>
   <extends>QWidget</extends>
   <header>widgets/loginstatewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>check_interval</tabstop>
  <tabstop>download_dir</tabstop>
  <tabstop>download_dir_browse</tabstop>
  <tabstop>auto_download</tabstop>
  <tabstop>delete_after</tabstop>
  <tabstop>username</tabstop>
  <tabstop>password</tabstop>
  <tabstop>device_name</tabstop>
  <tabstop>login</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
